zerver/templatetags/minified_js.py

```diff
@@ -24,7 +24,7 @@ def render(self, context: Dict[str, Any]) -> str:
         else:
             scripts = [settings.JS_SPECS[self.sourcefile]['output_filename']]
         script_urls = [staticfiles_storage.url(script) for script in scripts]
-        script_tags = [('<script type="text/javascript" nonce="%s"'
-                       ' src="%s" charset="utf-8"></script>') % (self.csp_nonce, url)
```

Comment:
Are these attributes actually invalid?  See https://www.w3schools.com/tags/att_script_charset.asp

Reply:
I'd say they are valid attributes for `<script>` tag but actually not needed, unless your javascript is actually needs special character from `UTF-8`, `UTF-16` or `ISO-*`.

In my experience developers never actually need to define those attributes.

Reply:
In general, I recommend ignoring w3schools in favor of MDN and the actual standards, which provide much more accurate and comprehensive information. The standard lists `<script charset>` as [“obsolete but conforming” and “should not specify”](https://html.spec.whatwg.org/multipage/obsolete.html#obsolete), and has no effect since it [defaults to the document encoding](https://html.spec.whatwg.org/multipage/scripting.html#script-processing-encoding), which is `utf-8` as it [must be](https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-charset) anyway.